static-analysis/teammates-checkstyle.xml

```diff
@@ -173,6 +173,27 @@
       <property name="onCommentFormat" value="CHECKSTYLE\.ON:([A-Za-z\|]+)"/>
       <property name="checkFormat" value="$1"/>
     </module>
+    <module name="FallThrough"/>
+    <module name="ArrayTrailingComma"/>
+    <module name="IllegalThrows">
+      <property name="ignoredMethodNames" value="execute"/>
+      <!-- Remove execute value from this property after upgrading to JUnit 5 (see teammates.test.cases.Executable) -->
```

Comment:
If this is what you want, then you should put `CHECKSTYLE.OFF` around that method only, not suppressing for all methods named `execute`.